package-lock.json

```diff
@@ -19239,6 +19243,7 @@
     "ui-manchette": {
       "name": "@osrd-project/ui-manchette",
       "version": "0.0.1-dev",
+      "extraneous": true,
```

Comment:
Hm, why are these still mentioned here?

Maybe storybook dependencies still refer to the old packages?

Reply:
Hm I don't know, I tried to npm install again, but the dependencies are still there (and they're not mentioned in stoybook)